wiki/Rules/Code_of_conduct_for_modding_and_mapping/vi.md

```diff
@@ -0,0 +1,60 @@
+# Quy tắc ứng xử với modding và mapping cho cộng tác viên
+
+**Quy tắc ứng xử đối với modding và mapping** là quy tắc và cũng là hướng dẫn áp dụng cho toàn bộ cộng đồng [modding](/wiki/Modding) và [mapping](/wiki/Beatmapping) cũng như phần mở rộng cho [Quy tắc cộng đồng osu!](/wiki/Rules) và [Tiêu chí xếp hạng chung](/wiki/Ranking_criteria). Để đảm bảo rằng các cuộc thảo luận mapping và modding ở các chủ đề diễn ra trong môi trường mang tính xây dựng, tích cực và hiệu quả, quy tắc ứng xử là rất quan trọng để mọi người ở cùng một chủ đề. Khi tham gia vào cộng đồng sửa đổi và lập bản đồ, bắt buộc phải tuân theo quy tắc ứng xử này. Hành vi sai trái vi phạm các quy tắc này có thể dẫn đến [hình phạt đối với tài khoản của bạn](/wiki/Silence).
```

Comment:
**Quy tắc ứng xử đối với modding và mapping** là những quy tắc, hướng dẫn được áp dụng cho toàn bộ cộng đồng [modding](/wiki/Modding) và [mapping](/wiki/Beatmapping), cũng như là phần mở rộng cho [Quy tắc cộng đồng osu!](/wiki/Rules) và [Tiêu chí xếp hạng chung](/wiki/Ranking_criteria). Để đảm bảo rằng các cuộc thảo luận về mapping và modding ở các chủ đề diễn ra trong môi trường mang tính xây dựng, tích cực và hiệu quả, các quy tắc ứng xử là rất quan trọng để mọi người ở cùng một chủ đề. Khi tham gia vào cộng đồng modding và mapping, bắt buộc phải tuân theo các quy tắc ứng xử này. Hành vi vi phạm các quy tắc này có thể dẫn đến [hình phạt đối với tài khoản của bạn](/wiki/Silence).